hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/MergeOnReadIncrementalRelation.scala

```diff
@@ -59,6 +62,16 @@ class MergeOnReadIncrementalRelation(sqlContext: SQLContext,
     super.timeline.findInstantsInRange(startTimestamp, endTimestamp)
   }
 
+  override def buildScan(requiredColumns: Array[String], filters: Array[Filter]): RDD[Row] = {
```

Comment:
@xiarixiaoyao we don't need to override `buildScan` for that you can do the same thing w/in `composeRDD`

Reply:
ok， let me fixed it